services/py-api/src/database/repository/teams_repository.py

```diff
@@ -57,3 +57,6 @@ async def update(
 
     async def delete(self, obj_id: str, session: Optional[AsyncIOMotorClientSession] = None) -> Result:
         raise NotImplementedError()
+    
+    async def get_number_registered_teams(self):
+        return self._collection.find({"is_verified": "true"}).count()
```

Comment:
Same thing with this one. This is implemented in the latest commit of the main branch. So pulling the new changes from main should give you a function which gives you the of the teams. Looks like this and its in teams_repository.py, so you can just delete the function that you have written yourself as you wont need it

```
async def get_verified_registered_teams_count(self) -> int:
        """Returns the count of verified teams."""
        # Ignoring mypy type due to Returning Any from function declared to return "int"  [no-any-return] which is not true
        return await self._collection.count_documents({"is_verified": True})  # type: ignore
```